### ISSUE TITLE
[BUG] Radar Relay Decimal Operation

### ISSUE THREAD

**Issue by JoaoCampos89** [CoinAlpha/hummingbot#1020]
**Describe the bug**

Radar Relay tests not running, giving errors on decimals operations

**Steps To Reproduce**

Run Radar Relay tests

Here are the steps to reproduce the issue (see attachments in the section below):

Replace tests with following operations to work
- test_single_limit_order_cancel
line 180        price=current_price - Decimal("0.2") * current_price,
 - test_limit_buy_and_sell_and_cancel_all
line  203                                price=current_price - Decimal("0.2") * current_price,

- test_order_expire
line 239   price=current_price - Decimal("0.2") * current_price,

 - test_orders_saving_and_restoration
line 319   bid_price: float = current_bid_price * Decimal("0.8")
line 322   amount: float = Decimal("0.00005") / bid_price
- test_order_fill_record
line 399  amount: Decimal = Decimal("0.00005") / current_price

*WARNING: Do NOT publish any exchange API keys or your wallet's private key. Whoever has access to them may steal your assets!*

**Optional: your discord username:**
**Optional: your ETH address:** (If you want to participate in the bug bounty)

**Comment by rennel-tabing** [comment]
@JoaoCampos89 
We will need the following information for us to further check on the issue.
```
1) Discord username of user

2) Hummingbot version

3) Deployment environment (local/cloud, which cloud provider, which operating system, docker or from source)

4) Screenshot: In the hummingbot client, run the status command and take a screenshot that includes the full Terminal/Bash window. Make sure to include both the left pane and the right log pane of Hummingbot.

5) Strategy config file: This is the file with the parameters associated with your strategy. It does not contain API keys, wallet private keys, or other confidential data. It is located in the conf/ folder. For example, if you have configured a pure market making strategy for the first time, the file's default name is conf_pure_market_making_0.yml

6) Log file: This is the file that contains a detailed log output and error stack trace. It is located in the logs/ folder. If you are sending logs related to your most recent Hummingbot session, sort the folder for the most recently updated file. It should have a name similar to hummingbot_logs_2019-07-31-19-51-36.log
```

**Comment by JoaoCampos89** [comment]
@rennel-tabing this is on tests folder no need for the strategy config file, my discord username is Joao Campos | VeriSafe Team#3218

**Comment by dennisocana** [comment]
@veecos2 for the failing unit tests in Radar Relay market.

**Comment by martinkou** [comment]
Fixed with #1089.
